usaspending_api/references/tests/integration/filter_tree/conftest.py

```diff
@@ -0,0 +1,22 @@
+from usaspending_api.references.tests.integration.filter_tree.tas.tas_data_fixtures import (
+    basic_agency,
+    cfo_agencies,
+    non_cfo_agencies,
+    unsupported_agencies,
+    multiple_federal_accounts,
+    agency_with_unsupported_fa,
+    multiple_tas,
+    fa_with_unsupported_tas,
+)
```

Comment:
Is there a value in doing it this way over just putting all the fixtures in conftest.py?  It seems like unnecessary work to put all the fixtures in one file and then import them into another.  I suppose if we ever have multiple files, but we can burn that bridge when we get to it.

Reply:
I'd love to hear your alternative. I hold no love for these conftest files in the first place; I just put up with them as a nessicary step for the otherwise quite nice pytest fixture patter.

Reply:
Oh, I was just suggesting copying the contents of tas_data_fixtures.py into this file and deleting tas_data_fixtures.py.  You don't have to, though.  I was just wondering if there was some value I wasn't seeing doing it this-a-way.

Reply:
It does seem unnecessary, curious to learn the benefit